Makefile

```diff
@@ -86,3 +101,41 @@ fix-lint: ## Automatically fix linting issues where possible
 clean: ## Run go clean and remove the rollout-operator binary
 	rm -f rollout-operator
 	go clean ./...
+
+.PHONY: check-jsonnet-manifests
+check-jsonnet-manifests: ## Check the rollout-operator manifests.
+check-jsonnet-manifests: format-jsonnet-manifests
+	@echo "Checking diff:"
+	@./tools/find-diff-or-untracked.sh $(JSONNET_MANIFESTS_PATHS) || (echo "Please format jsonnet manifests by running 'make format-jsonnet-manifests'" && false)
+
+.PHONY: format-jsonnet-manifests
+format-jsonnet-manifests: ## Format the rollout-operator manifests.
+	@find $(JSONNET_MANIFESTS_PATHS) -type f -name '*.libsonnet' -print -o -name '*.jsonnet' -print | xargs $(JSONNET_FMT) -i
+
+.PHONY: build-jsonnet-tests
+build-jsonnet-tests: ## Build the rollout-operator tests.
+	@./operations/rollout-operator-tests/build.sh
+
+.PHONY: check-jsonnet-tests
+check-jsonnet-tests: ## Check the jsonnet tests output.
+check-jsonnet-tests: build-jsonnet-tests jsonnet-conftest-test
+	@./tools/find-diff-or-untracked.sh operations/mimir-tests || (echo "Please rebuild jsonnet tests output 'make build-jsonnet-tests'" && false)
+
+.PHONY: jsonnet-conftest-quick-test
+jsonnet-conftest-quick-test: ## Does not rebuild the yaml manifests, use the target jsonnet-conftest-test for that
+jsonnet-conftest-quick-test:
+	@./operations/rollout-operator-tests/run-conftest.sh --policies-path $(REGO_POLICIES_PATH) --manifests-path ./operations/rollout-operator-tests
+
+.PHONY: jsonnet-conftest-test
+jsonnet-conftest-test: build-jsonnet-tests jsonnet-conftest-quick-test
+
+.PHONY: check-makefiles
+check-makefiles: ## Check the makefiles format.
+check-makefiles: format-makefiles
+	echo $(MAKE_FILES)
+	@git diff --exit-code -- $(MAKE_FILES) || (echo "Please format Makefiles by running 'make format-makefiles'" && false)
+
+.PHONY: format-makefiles
+format-makefiles: ## Format all Makefiles.
+format-makefiles: $(MAKE_FILES)
+	$(SED) -i -e 's/^\(\t*\)  /\1\t/g' -e 's/^\(\t*\) /\1/' -- $?
```

Comment:
This seems like an unrelated change, I would move this to a separate PR (and also add it to the CI workflow)